Guideline.md

```diff
@@ -282,6 +281,12 @@ At the event, make sure there are an appropriate number of sober staff on hand t
 #### 9.1 Name badges
 The purpose of a name badge is to identify people easily. As such, it should be easy to read! You should use a large, easy to read font. Sans serif fonts are generally regarded as a good choice. Make the name stand out and contrast well - avoid grey on grey! A font size of 18 points or bigger if you can make it fit would make a difference.
 
+Also consider allowing people to define or even write in their prefered name, rather than simply printing their legal name and do not require photo ID but rather use a QR code send via email to allow people to identify themselves. Furthermore allow space for people to define their preferred pronouns or [provide premade buttons/stickers](http://yupgup.com/pronouns.html).
+
+Additionally it may make sense to provide stickers for people to denote things like "first time attendee", "ask me about [ - space to write in- ]", "talk to me" to forster more interaction between attendees.
+
+Finally provide stickers or differently colored lanyards to enable people to choose if they want photos taken of themselves and how these may be used. For inspiration see [here](https://twitter.com/imeluny/status/956072960296673280?lang=en).
```

Comment:
The word "Finally" in a living document can create bugs. If providing differently coloured lanyards, take colour blindness into account by also possibly having different patterns on them.